src/sqsd.groovy

```diff
@@ -68,8 +68,8 @@ try {
         def messages = sqs.receiveMessage(receiveMessageRequest).getMessages()
         println "Received Messages : " + messages.size()
 
-        // Break when empty
-        if(messages.size() <= 0) break
+        // Break when empty if not running daemonized
+        if(messages.size() <= 0 && config.sqsd.run_daemonized < 1) break
```

Comment:
My reservation around that is it adds unnecessary latency when long-polling is working correctly. 

I'll add it in though since I suspect most would prefer using this in a non-daemonized cron job in a more prod-like setting.